packages/compartment-mapper/src/compartment.d.ts

```diff
@@ -0,0 +1,94 @@
+// https://github.com/tc39/proposal-compartments b420786 on Jul 6
```

Comment:
These types belong in SES. Do they need to be here for this change to work?

Reply:
This _is_ SES, no? Oh... I guess this monorepo has another package for SES... yes, I suppose it belongs there.

Move it within this PR or make a separate PR?

Reply:
Within this PR unless there’s an obstacle. Do you depend upon these definitions here yet?